core/src/main/java/org/infinispan/reactive/publisher/impl/commands/reduction/ReductionPublisherRequestCommand.java

```diff
@@ -96,43 +141,13 @@ public byte getCommandId() {
    }
 
    @Override
-   public void writeTo(ObjectOutput output) throws IOException {
-      output.writeObject(getOrigin());
-      output.writeBoolean(parallelStream);
-      MarshallUtil.marshallEnum(deliveryGuarantee, output);
-      output.writeObject(segments);
-      MarshallUtil.marshallCollection(keys, output);
-      MarshallUtil.marshallCollection(excludedKeys, output);
-      output.writeLong(explicitFlags);
-      output.writeBoolean(entryStream);
-      if (transformer == finalizer) {
-         output.writeBoolean(true);
-      } else {
-         output.writeBoolean(false);
-         output.writeObject(transformer);
-      }
-
-      output.writeObject(finalizer);
+   public int getTopologyId() {
```

Comment:
This must be sent over the network too.

Reply:
Good spot, I missed the `TopologyAffectedCommand` interface.